R1 reduce jargon & repitition (#297)
SORTEE-Github-Hackathon/manuscript: content/03.use-cases-a.md

The ability to make a GitHub repository private is particularly advantageous for storing materials without necessarily sacrificing privacy, which may be preferred or even essential for certain compendia. 
Researchers can then grant specific collaborators read and/or write access to private repositories to pursue pre-publication analyses or writing in privacy, or even to collaborate on research that will permanently remain private.

> <!-- Is this paragraph too complicated/jargony?-->

Reviewer: I think it's fine

Reviewer 2: I think the first sentence is hard to follow, but could easily be remedied by changing the sentence structure a bit e.g. "Additionally, users have the ability to make their repository private, allowing researchers to store materials without sacrificing privacy which may be preferred, or essential for certain subject matter"

> GitHub limits committed file sizes to 100 Mb (megabytes) [@url:https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github], which can make it challenging for centralizing research compendia containing larger file sizes. 

Reviewer 3: ```suggestion
GitHub limits committed file sizes to 100 Mb (megabytes) [@url:https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github], which can make it challenging to use as a repository for _all_ research files. 
```

Removing jargon

> Users may still version large files using Git Large File Storage (<https://git-lfs.github.com>) text pointers, but may have to rely on external file storage alternatives (such as local or cloud-hosting).

Reviewer 3: ```suggestion
Users may still version large files using Git Large File Storage (<https://git-lfs.github.com>), but may have to rely on external file storage alternatives (such as local or cloud-hosting).
```

For instance, one can create a discussion on a project repository to decide which method to apply for biodiversity data analysis.
Then, an issue can be created to establish steps and responsibilities including data formatting, statistical analyses, figure generation, and issue resolution [_e.g._, see issues and discussions in the sPlotOpen manuscript repository, @doi:10.1111/geb.13346]; @url:https://github.com/fmsabatini/sPlotOpen_Manuscript/issues?q=is%3Aissue+is%3Aclosed].
Using GitHub for all project-related conversation and planning, rather than email or messaging tools, makes it easier to keep track of progress throughout the lifespan of a project. 
However, one can opt to receive new issues, discussions, and responses as emails and can post replies by email as well.  
This allows for centralized communication for a team even when some members prefer to use email for communication.
Unlike emails and messages which can get lost as more new tasks arise, GitHub issues are intentionally closed by repository administrators hiding the issue from view (closed issues remain accessible but not immediately visible).

> <!-- ERS: I removed the Zenhub / Slack section as it seemed like it might fall into R1's definition of "too nerdy". Plus, new GitHub projects has basically all the features of Zenhub-->

Reviewer 3: Sounds fine to me.

The entire process of running a course, workshop, or lecture, can all be done openly on GitHub including material development, web hosting, and delivery, and even submission and grading of assignments.
While there are other purpose-built platforms for this, GitHub provides a free, open-source alternative.

<!-- remove/simplify stuff about tools to make documents.  out of scope -->
Syllabi, presentations, and other course materials can be version-controlled and stored in GitHub. 
Although plain text formats such as markdown or HTML work best with GitHub and help to keep the total size of the repository within limits, any file type can be stored on and downloaded from GitHub including PowerPoint and Word documents.
Content can then be delivered by sharing a link to the GitHub repository or creating a template repository with assignments that students can use.
Using GitHub to host a course website is also an option for making course content more widely and easily accessible (see [Hosting a Website](#hosting-a-website)).

> Using GitHub to collect student submissions is perhaps the most challenging component, but the new GitHub Classroom tool (<https://classroom.github.com>) allows instructors to host private assignments to be submitted collaboratively or individually as code or PDF files, and even build autograding tests.

Reviewer: maybe change "the most challenging component" to "a more challenging component"

Reviewer 3: I added @fhillemann's suggestion in review comment below

Author: I think I'll go with something like "a more advanced use".

> Using GitHub to collect student submissions is perhaps the most challenging component, but the new GitHub Classroom tool (<https://classroom.github.com>) allows instructors to host private assignments to be submitted collaboratively or individually as code or PDF files, and even build autograding tests.

Reviewer 3: ```suggestion
Using GitHub to collect student submissions is perhaps a more challenging component, but the new GitHub Classroom tool (<https://classroom.github.com>) allows instructors to host private assignments to be submitted collaboratively or individually as code or PDF files, and even build autograding tests.
```
